src/english.yml

```diff
@@ -577,3 +577,13 @@ msg_err_friendly_fire_disable: 'Friendly fire is disabled!'
 
 # Added in 0.38
 msg_claim_error: '&cAnother plugin stopped the claim of (%s)/(%s) town blocks, could not complete the operation.'
+
+
```

Comment:
"# Added in 0.39" should be put in front of this addition and the language file should be bumped. I will take care of the other languages.